cluster-install/install.sh

```diff
@@ -2443,12 +2486,13 @@ step_monitor_deployment_progress() {
         fi
     fi
 
+    local -r torc_name="${_EXISTING_TORC_NAME:-"$__DEFAULT_TORC_NAME"}"
```

Comment:
Was noticing that the monitoring function was failing the TORC status check sometimes. Turns out that `_EXISTING_TORC_NAME` is empty when, well, there's no existing TORC (duh). So we now fall back to the default TORC name (used for fresh installs) in those cases.